CogniteSdk.Types/Beta/Subscriptions/SubscriptionDataResponse.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright 2023 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Collections.Generic;
+
+namespace CogniteSdk.Beta
+{
+    /// <summary>
+    /// The ID of a single timeseries that has been updated.
+    /// </summary>
+    public class TimeSeriesId
```

Comment:
```suggestion
    public class WrappedTimeSeriesId
```
maybe fits a bit better?

Reply:
Sure